compiler/rustc_trait_selection/src/traits/error_reporting/suggestions.rs

```diff
@@ -1351,13 +1352,34 @@ impl<'tcx> TypeErrCtxtExt<'tcx> for TypeErrCtxt<'_, 'tcx> {
                         );
                     } else {
                         let is_mut = mut_ref_self_ty_satisfies_pred || ref_inner_ty_mut;
+
+                        // see #104961
```

Comment:
Could you elaborate the comment? Links to issues make understanding harder than a sentence describing the issue.